### PR TITLE
Remarks from Olivier on crowdfunding website

### PR DESCRIPTION
**Pull request by quentingigon**

### CODE REVIEW

File: crowdfunding_compassion/controllers/projects_controller.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ def get_projects_list(self, **kwargs):
            auth="public",
            type="http",
            method='POST',
+           csrf=False,
```
Comment (ecino) [inline crowdfunding_compassion/controllers/projects_controller.py]: Please remove this, it will introduce security issues. It works fine on the devel server.

File: crowdfunding_compassion/forms/project_creation_form.py [hunk 2]
```diff
@@ -213,7 +213,7 @@ def _form_fieldsets(self):
             ],
         }]
         storage = self.wiz_storage_get().get('steps')
-        if self.main_object or storage.get(2, {}).get("type") == "collective":
+        if storage.get(1, {}).get("type") == "collective" or self.main_object:
```
Comment (ecino) [inline crowdfunding_compassion/forms/project_creation_form.py:216]: I don't think this is a good change, we need values of the step1 form and not step2. Why taking the storage in priority to the main_object ? I think you should revert this change.

File: crowdfunding_compassion/static/src/js/creation_form.js [hunk 3]
```diff
@@ -62,3 +62,8 @@ odoo.define('crowdfunding_compassion.creation_form', function (require) {
 
     return animation.registry.crowdfunding_creation_form;
 });
+
+$( document ).ready(function() {
+    presentation_input = document.getElementById("presentation_video")
+    presentation_input.placeholder = "e.g. https://vimeo.com/jlkj34ék5"
+});
```
Comment (ecino) [inline crowdfunding_compassion/static/src/js/creation_form.js]: This can be put in the attribute `help` of the field `presentation_video` in the model `crowdfunding.participant` and `crowdfunding.project`
```suggestion
```

**Review by ecino — CHANGES_REQUESTED** [review]

File: crowdfunding_compassion/models/crowdfunding_project.py [hunk 4]
```diff
@@ -39,7 +39,7 @@ class CrowdfundingProject(models.Model):
         help="Upload a cover photo that represents your project. Best size: 900x400px",
         attachment=True)
     presentation_video = fields.Char(
-        help="Paste any video link that showcase your project (Youtube or Vimeo)"
+        help="https://vimeo.com/jlkj34ék5"
```
Comment (ecino) [inline crowdfunding_compassion/models/crowdfunding_project.py]:
```suggestion
        help="Paste any video link that showcase your project (e.g. https://vimeo.com/jlkj34ek5)"
```

File: crowdfunding_compassion/models/crowdfunding_participant.py [hunk 5]
```diff
@@ -31,7 +31,7 @@ class CrowdfundingParticipant(models.Model):
     )
     source_id = fields.Many2one('utm.source', 'UTM Source',
                                 required=True, ondelete='cascade')
-    presentation_video = fields.Char(help="Youtube/Vimeo link")
+    presentation_video = fields.Char(help="https://vimeo.com/jlkj34ék5")
```
Comment (ecino) [inline crowdfunding_compassion/models/crowdfunding_participant.py]:
```suggestion
        help="Paste any video link that showcase your project (e.g. https://vimeo.com/jlkj34ek5)"
```

**Review by ecino — CHANGES_REQUESTED** [review]

File: crowdfunding_compassion/forms/donation_form.py [hunk 6]
```diff
@@ -71,7 +71,7 @@ def generate_invoice(self):
                             0,
                             0,
                             {
-                                "quantity": self.amount / product.standard_price,
+                                "quantity": self.amount / product.standard_price if product.standard_price != 0 else 0,
```
Comment (ecino) [inline crowdfunding_compassion/forms/donation_form.py]:
```suggestion
                                "quantity": self.amount / product.standard_price
                                if product.standard_price != 0 else 0,
```

**Review by ecino — CHANGES_REQUESTED** [review]

File: crowdfunding_compassion/models/crowdfunding_project.py (hunk 4 above)
Comment (quentingigon) [reply crowdfunding_compassion/models/crowdfunding_project.py]: Oups

**Review by ecino — APPROVED** [review]
